content/outcomes/cloud-native-development/specific__caching.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Caching
+weight: 120
+layout: single
+team:
+ - VMware Tanzu Labs
+---
+
+Caching can be made available in many forms, from large vendor-specific externalized clusters to caching libraries that cache in-memory within the application itself. While moving to a platform such as Cloud Foundry (TAS) or Kubernetes, caching's most prominent interaction is with the elastic and ephemeral nature of the application deployment. Application deployments can come and go at random and scale from zero to hundreds of instances. It conditions the way applications cache data.
```

Comment:
Consider describing and referencing Spring alternatives.

Reply:
Done! ce88ac4e39ecd0d9608c15e68132693bf7de7ff8